testing/apps/details/Dockerfile.sidecar

```diff
@@ -19,4 +19,4 @@ COPY details /opt/microservices/
 
 EXPOSE 9080
 
-ENTRYPOINT ["a8sidecar", "/opt/microservices/details", "9080" ]
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/dumb-init", "--", "a8sidecar", "/opt/microservices/details", "9080" ]
```

Comment:
This is not directly related to this PR, but the bookinfo/helloworld Dockerfiles shouldn't redefine the ENTRYPOINT, but rather inherit it from the base image, and add their arguments as CMD:

```
CMD [ "/opt/microservices/details", "9080" ]
```